app/models/profile_validation_decision.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class ProfileValidationDecision < ApplicationRecord
+  has_paper_trail
+
+  validates :note, presence: true
+  validates :approved, inclusion: { in: [true, false] }
```

Comment:
What other values can there be for what I assume is a boolean?

Reply:
Does this mean that it's not possible to leave a note without making a decision?

Reply:
It rejects `nil`. As for now it is not possible to leave a comment without making a decision.